lib/resources/members.js

```diff
@@ -143,6 +143,85 @@ const removeFromWishlist = (programCode, memberUUID) => {
   };
 };
 
+/**
+ *
+ * @param {string} programCode pangea program code
+ * @param {string} memberUUID PII identificator
+ * @returns {*} function
+ */
+const getShoppingCart = (memberUUID, programCode) => {
+  /**
+   *
+   * @param {object} params
+   * @param {module:api~requestCallback} callback Callback that handles the response
+   */
+  return function shoppingCart(params, callback) {
+    if (typeof params !== 'object' || isEmpty(params)) {
+      if (arguments.length === 1 && typeof params === 'function') {
+        callback = params;
+      }
+
+      callback(new Error('A params object is required'));
+
+      return;
+    }
+
+    if (typeof params === 'object') {
+      if (!params.segment_code) {
+        callback(new Error('A segment_code param is required'));
+
+        return;
+      }
+    }
```

Comment:
How much error handling should we do in the SDK? Is it best to just let Pangea errors come through? That's what would happen otherwise, right?

Reply:
I believe so, yes... also that's what we're doing with all other ones as well, so I think we can leave it at that.

Reply:
Ok, but don't remove this check for segment_code, for example?

Reply:
yep.